source/features/link-to-github-io.tsx

```diff
@@ -30,15 +30,21 @@ function initRepo(signal: AbortSignal): void {
 }
 
 function initRepoList(signal: AbortSignal): void {
-	observe('a[href$=".github.io"][itemprop="name codeRepository"]', addRepoListLink, {signal});
+	// Earlier GitHub Pages were hosted on github.com #6228
+	observe(`
+		a[itemprop="name codeRepository"]:is(
+			[href$=".github.io"],
+			[href$=".github.com"]
+		)
+	`, addRepoListLink, {signal});
```

Comment:
Interestingly this broke our selector parser https://github.com/g-plane/typed-query-selector/issues/26

Let's just use an array

```suggestion
	observe([
		// Earlier GitHub Pages were hosted on github.com #6228
		'a[itemprop="name codeRepository"][href$=".github.com"]',
		'a[itemprop="name codeRepository"][href$=".github.io"]',
	], addRepoListLink, {signal});
```